appd.adoc

```diff
@@ -112,15 +85,15 @@ or
 ----
 p(n,k,j,i) = ap(k) + b(k)*ps(n,j,i)
 ----
+
 where **`p(n,k,j,i)`** is the pressure at gridpoint **`(n,k,j,i)`**, **`a(k)`** or **`ap(k)`** and **`b(k)`** are components of the hybrid coordinate at level **`k`**, **`p0`** is a reference pressure, and **`ps(n,j,i)`** is the surface pressure at horizontal gridpoint **`(j,i)`** and time **`(n)`**.
 The choice of whether **`a(k)`** or **`ap(k)`** is used depends on model formulation; the former is a dimensionless fraction, the latter a pressure value.
 In both formulations, **`b(k)`** is a dimensionless fraction.
 
-The format for the **`formula_terms`** attribute is
-----
-
-formula_terms = "a: var1 b: var2 ps: var3 p0: var4"
+The format for the **`formula_terms`** attribute is 
```

Comment:
Not sure why this one is in bold as well as monospace? For consistency with the rest of the rendered document:

```suggestion
The format for the `formula_terms` attribute is 
```

Reply:
But isn't the bold version there right one? There are 33 occurrences of \*\*\`formula_terms\`\*\* and only 5 of \`formula_terms\`, and those 5 don't appear to be unique situations.

?

Reply:
Fair point! Consistency is key in my eyes, but there's consistency for the `formula_terms` references, and consistency over any variable references, too. I think it is breaking the latter, but haven't checked carefully. Maybe this needs discussion, first.